bin/node/runtime/src/lib.rs

```diff
@@ -986,6 +1001,28 @@ impl_runtime_apis! {
 		fn current_epoch_start() -> sp_consensus_babe::SlotNumber {
 			Babe::current_epoch_start()
 		}
+
+		fn generate_key_ownership_proof(
+			authority_id: sp_consensus_babe::AuthorityId,
```

Comment:
As with grandpa, we probably want to include the `SessionIndex` or parent hash here

Reply:
or slot number would be the most domain-specific sensible example